packages/kit/test/apps/options/test/test.js

```diff
@@ -10,7 +10,7 @@ test.describe('base path', () => {
 		const response = await request.get('/slash/', { headers: { Accept: 'text/html' } });
 		expect(response.status()).toBe(404);
 		expect(await response.text()).toBe(
-			'The server is configured with a public base URL of /path-base - did you mean to visit <a href="/path-base/slash/">/path-base/slash/</a> instead?'
+			'Not found (did you mean <a href="/path-base/slash/">/path-base/slash/</a>?)'
```

Comment:
I probably would have changed the message to align with Vite instead as it'd make it easier to eventually switch over to their handling of things

Reply:
The language doesn't quite mesh with our usage though — if we wanted to be similarly verbose we'd probably have something like

> The app is configured with a `paths.base` of /path-base/ — did you mean to visit ...

so that it refers to the config value as exposed to Kit users, and includes the trailing slash. I think it's better to do our own thing here